app/models/wind_turbine_vibration_measurement.rb

```diff
@@ -0,0 +1,15 @@
+class WindTurbineVibrationMeasurement < ApplicationRecord
```

Comment:
Speed is the only field of the WindTurbineSeedMeasurement model. Could we use only one model/table called WindTurbineMeasurement and include there all the fields related to the WindTurbine?